README.md

```diff
@@ -52,7 +53,7 @@ Router](https://www.silabs.com/documents/public/application-notes/an1256-using-s
   * SL-OPENTHREAD/2.2.3.0_GitHub-91fa1f455 ([release notes](https://www.silabs.com/documents/public/release-notes/open-thread-release-notes-2.2.3.0.pdf))/
   * SL-OPENTHREAD/2.3.0.0_GitHub-dae3ff2c5 ([release notes](https://www.silabs.com/documents/public/release-notes/open-thread-release-notes-2.3.0.0.pdf))/
   * SL-OPENTHREAD/2.3.1.0_GitHub-e6df00dd6 ([release notes](https://www.silabs.com/documents/public/release-notes/open-thread-release-notes-2.3.1.0.pdf))/
-
+  * SL-OPENTHREAD/2.3.2.0_GitHub-e6df00dd6 ([release notes](https://www.silabs.com/documents/public/release-notes/open-thread-release-notes-2.3.2.0.pdf))/
```

Comment:
this hash cant be right? but its what is in the document so 🤷

Reply:
It's correct. It just means that the OpenThread version for this Gecko SDK release uses the exact same version as the previous one.